blueprints/data-solutions/gcs-to-bq-with-least-privileges/variables.tf

```diff
@@ -42,18 +43,18 @@ variable "prefix" {
   }
 }
 
-variable "project_create" {
-  description = "Provide values if project creation is needed, uses existing project if null. Parent is in 'folders/nnn' or 'organizations/nnn' format."
+variable "project_config" {
+  description = "Provide 'billing_account_id' value if project creation is needed, uses existing 'project_id' if null. Parent is in 'folders/nnn' or 'organizations/nnn' format."
```

Comment:
Maybe worth mentioning, that if the project will be created, `var.prefix` will be used (contrary to using existing project)